.yarn/patches/@lavamoat-lavapack-npm-3.3.0-5a61b5374d.patch

```diff
@@ -36,10 +36,10 @@ index 97c9ddb9b76a2fe31cf9b202cb491f15232232ff..f7f66db9e16d58e2f50bae9491eae88b
          })
        })
 @@ -11556,6 +11558,17 @@ module.exports = {
- 
- 
+
+
    function loadModuleData (moduleId) {
-+    if (typeof window === 'undefined' && require('node:module').isBuiltin(moduleId)) {
++    if (typeof window === 'undefined' && typeof require === 'function' && require('node:module').isBuiltin(moduleId)) {
```

Comment:
This patch is required for inlining the LavaMoat bundle in webworkers. Without the check for `require`, it would result in an error saying that `require` is not defined, because we assume it's run in a Node.js environment.